geoportailv3/static/js/themeservice.js

```diff
@@ -7,16 +7,42 @@ goog.require('app.Themes');
 goog.require('ngeo.Location');
 
 
+/**
+ * @typedef {{push: function(Array<string>)}}
+ */
+app.Piwik;
+
+
 
 /**
  * @constructor
+ * @param {angular.$window} $window Global Scope.
  * @param {ngeo.Location} ngeoLocation ngeo Location service.
  * @param {app.Themes} appThemes
  * @param {app.ScalesService} appScalesService Service returning scales.
  * @param {Array.<number>} maxExtent Constraining extent.
  * @ngInject
  */
-app.Theme = function(ngeoLocation, appThemes, appScalesService, maxExtent) {
+app.Theme = function($window, ngeoLocation, appThemes,
+    appScalesService, maxExtent) {
+
+  /**
+   * @const
+   * @private
+   * @type {Object}
+   */
+  this.piwikSiteIdLookup_ = {
+    'main': 18,
+    'tourisme': 7,
+    'emwelt': 8,
+    'eau': 6
```

Comment:
for the different ids: we wont keep tourismev2 so no need

for the complete list i agree but we can also add it later